Aquality.Selenium/src/Aquality.Selenium/Browsers/AqualityServices.cs

```diff
@@ -88,11 +106,13 @@ public static void SetDefaultFactory()
         /// <typeparam name="T">type of required service.</typeparam>
         /// <exception cref="InvalidOperationException">Thrown if there is no service of the required type.</exception> 
         /// <returns></returns>
-        public static T GetRequiredService<T>()
+        public static T Get<T>()
```

Comment:
I thought you decided to make it private

Reply:
not exactly, we agreed make private ServiceProvider